decorators2.py

```diff
@@ -0,0 +1,26 @@
+
+def surprise_decorator(function):
+    """
+    This is the decorator that calls the wrapper which call the function that prints "Surprise".
+    :param function: the function we want to call (in this case- surprise()).
+    :return: the value of the wrapper which is the function that prints "Surprise".
+    """
+    def surprise_wrapper():
+        """
+        This wrapper calls (when returning) the function we passed which in this case is surprise().
+        :return: the function which in this case is surprise().
+        """
+        return function()
```

Comment:
The decorator should print "Surprise!" **instead** of executing the original function.